service-front/module/Application/tests/View/DateCheckViewModelHelperTest.php

```diff
@@ -0,0 +1,324 @@
+                                    'section 9 of the LPA, or on the same day.'],
+            'expectedAttorneyText' => []
+        ],
+        // Combined CS1 and CS2
+        // 1. LPA has more than 4 people to notify (generates CS1)
+        // 2. additional information on when replacement attorneys should act (generates CS2)
+        [
+            'lpa' => [
+                'document' => [
+                    'peopleToNotify' => [[], [], [], [], []],
+                    'replacementAttorneyDecisions' => [
+                        'when' => 'When the primary attorney cannot be contacted'
+                    ]
+                ]
+            ],
+            'expectedDonorText' => ['You must have signed and dated continuation sheets 1 and 2 before you signed ' .
+                                    'section 9 of the LPA, or on the same day.'],
+            'expectedAttorneyText' => []
+        ],
+        // Continuation sheet 3
+        // Health & welfare LPA - donor cannot sign or make a mark
+        [
+            'lpa' => [
+                'document' => [
+                    'type' => 'health-and-welfare',
+                    'donor' => [
+                        'canSign' => false
+                    ]
+                ]
+            ],
+            'expectedDonorText' => ['This person must have signed continuation sheet 3 on the same day as they sign ' .
+                                    'section 5 and before the certificate provider signs section 10.'],
+            'expectedAttorneyText' => []
+        ],
+        // Property & finance LPA - donor cannot sign or make a mark
+        [
+            'lpa' => [
+                'document' => [
+                    'type' => 'property-and-financial',
+                    'donor' => [
+                        'canSign' => false
+                    ]
+                ]
+            ],
+            'expectedDonorText' => ['This person must have signed continuation sheet 3 before the certificate ' .
+                                    'provider has signed section 10.'],
+            'expectedAttorneyText' => []
+        ],
+        // Continuation sheet 4
+        // Property & finance LPA - primary attorney is a trust corporation
+        [
+            'lpa' => [
+                'document' => [
+                    'type' => 'property-and-financial',
+                    'primaryAttorneys' => [
+                        ['type' => 'corporation', 'number' => '123']
+                    ]
+                ]
+            ],
+            'expectedDonorText' => [],
+            'expectedAttorneyText' => ['They must have signed continuation sheet 4 after the ' .
+                                       '\'certificate provider\' has signed section 10 of the LPA form.']
+        ],
+    ];
+
+    /* This returns an arbitrary string to imitate a class constant needed when adding twig functions to the */
```

Comment:
PHP comments work like this:

```
/*
 * comment 
 * comment
 */
```

so we don't need to repeat the open/close character sequence on every line. Alternatively, use `//` on every line if you prefer.

Reply:
Thanks, I hadn't even noticed that, I'd been using the Vim shortcut and just assumed they were correct! Will have to dig into Vim config again :)